### PR TITLE
adding new 2trck cut

### PR DESCRIPTION
**Pull request by glromane**
Dear @victor-gonzalez ,

in this commit we add a new double track cut. Could you please approve it. Thank you.

Sincerely yours, Gleb.

### CODE REVIEW

File: PWGCF/Femto3D/Core/femto3dPairTask.h [hunk 1]
```diff
@@ -253,6 +260,24 @@ bool FemtoPair<TrackType>::IsClosePair(const float& deta, const float& dphi, con
...
+    return -100.f;
+
+  float dtheta = THETA(_first->eta()) - THETA(_second->eta());
+  float res = 0.0;
+
+  for (const auto& radius : TPCradii) {
+    res += sqrt(pow(2.0 * radius * sin(0.5 * GetPhiStarDiff(radius)), 2) + pow(2.0 * radius * sin(0.5 * dtheta), 2));
+  }
+
+  return 100.0 * res / 9;
```
Comment (victor-gonzalez) [inline PWGCF/Femto3D/Core/femto3dPairTask.h:278]: I would recommend using the length of the `TPCradii` instead of `9`

**Review by victor-gonzalez — DISMISSED** [review]
Have a look at my comment
I approve it but I will wait for merging it in case you want to modify it

File: PWGCF/Femto3D/Core/femto3dPairTask.h (hunk 1 above)
Comment (glromane) [reply PWGCF/Femto3D/Core/femto3dPairTask.h]: Dear @victor-gonzalez , thank you your comment. The size if fixed so I used just 9, now I changed it to std::array and calling the size() method. Is it OK like this?

Comment (victor-gonzalez) [reply PWGCF/Femto3D/Core/femto3dPairTask.h]: Thanks! It is OK
My point was that if someone in the future changes the size of the array by including more radii they only need to care about the array and nothing else

**Review by victor-gonzalez — APPROVED** [review]

File: PWGCF/Femto3D/Core/femto3dPairTask.h (hunk 1 above)
Comment (glromane) [reply PWGCF/Femto3D/Core/femto3dPairTask.h]: ah, I see. Thanks!

**Pull request merged**